apps/react-app/src/pods/manfred-json-input/manfred-json-input.component.tsx

```diff
@@ -8,18 +8,17 @@ interface Props {
 export const ManfredJsonInput: React.FunctionComponent<Props> = (props: Props) => {
 
   const {onSetManfredJson} = props;
+  const {macData, setMacData} = React.useContext(MacContext);
 
-  const [text, setText] = useState('');
```

Comment:
We should differentiate here two stages:
  - Stage 1: I'm entering some text that needs some validations (later on we will add schema validation), so.... it's just temporary content until the user hits on _next_ and the content is validated and promoted to _jsonManfredContent_

  - Stage 2: Cool the temporary text has been promoted to a valid Manfred content, now is time to store it in the context and move forward.

So:
- Keep the local text _useState_ 
- Once the user clicks on next, is not the responsibility of the presentational component to handle this, on the _onSetManfredJson_ we pass the content to the parent container, and the parent container will store the content in the context and navigate.


By following this approach we are adding a clear border:
- Hey UI Component you are responsible of showing data and ensure that the data is valid and trigger the next step.
- Hey Smart Component, here you go... cured data ready for the next step, smart guy go and do your job: store this in the context and navigate to the next step.